sqlalchemy_ibmi/base.py

```diff
@@ -1158,3 +1166,7 @@ def get_unique_constraints(self, connection, table_name, schema=None, **kw):
     def _check_text_server(self, connection):
         stmt = "SELECT COUNT(*) FROM QSYS2.SYSTEXTSERVERS"
         return connection.execute(stmt).scalar()
+
+    def do_executemany(self, cursor, statement, parameters, context=None):
+        cursor.fast_executemany = self.fast_executemany
```

Comment:
Could this not be once, set when the cursor is created?

Reply:
I figured because in the sqlalchemy code, there are instances where it creates a cursor and then closes it after the operation is complete, so setting it once would only set it in the first cursor.

Reply:
Ah, do we not have access to where it creates the cursor? I suppose not - they get a connection and then call methods on it. This seems like a good solution, then.